File: codewars/7kyu/allStarCodeChallenge3.js
/*
Create a function, called removeVowels (or remove_vowels), that takes a string argument and returns that same string with all vowels removed (vowels are "a", "e", "i", "o", "u").

removeVowels("drake") // => "drk"
removeVowels("aeiou") // => ""
*/

var removeVowels = function (str) {
    let output = '';
    for (let i = 0; i < str.length; i++) {
        if ((str[i] !== 'a') && (str[i] !== 'e') && (str[i] !== 'i') && (str[i] !== 'o') && (str[i] !== 'u')) output += str[i];
    };
    return output;
}

File: codewars/7kyu/allStarCodeChallenge3.py
# Create a function that takes a string argument and returns that same
# string with all vowels removed (vowels are "a", "e", "i", "o", "u").

# Example (Input --> Output)

# "drake" --> "drk"
# "aeiou" --> ""
# remove_vowels("drake") // => "drk"
# remove_vowels("aeiou") // => ""

def remove_vowels(strng):
    strng = strng.lower()
    output = ''
    for i in range(len(strng)):
        if strng[i] != 'a' and strng[i] != 'e' and strng[i] != 'i' and strng[i] != 'o' and strng[i] != 'u':
            output += strng[i]
    return output